doc/move_group_python_interface/scripts/move_group_python_interface_tutorial.py

```diff
@@ -342,13 +342,16 @@ def wait_for_state_update(
         ##
         ## Ensuring Collision Updates Are Received
         ## ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-        ## If the Python node dies before publishing a collision object update message, the message
+        ## If the Python node was just created and did not connect to other nodes yet, or
+        ## dies before publishing a collision object update message, the message
```

Comment:
I wasn't aware of this issue (anymore). Would be nice to add a reference:
```suggestion
        ## If the Python node was just created (https://github.com/ros/ros_comm/issues/176),
        ## or dies before actually publishing the scene update message, the message
```